bufutils/arrow.go

```diff
@@ -0,0 +1,19 @@
+package bufutils
+
+import (
+	"github.com/apache/arrow/go/v10/arrow"
+)
+
+// RecordSize returns the arrow record size in bytes.
+func RecordSize(r arrow.Record) int64 {
```

Comment:
nit: this might make more sense in the `arrowutils` package

Reply:
bah humbug I forgot about that